chapters/en/chapter5/fine-tuning-voxtral.mdx

```diff
@@ -0,0 +1,425 @@
+# Fine-tuning Voxtral for Speech Recognition
+
+In this section, we'll cover how to fine-tune Voxtral, Mistral AI's powerful speech recognition model, for automatic speech 
+recognition (ASR) on a new language. Voxtral is a state-of-the-art multimodal model that combines audio understanding with 
+the powerful language capabilities of the Mistral family.
```

Comment:
I would rephrase to make it clear that Voxtral is actually for more than just ASR.

Something like:
"
In this section, we'll cover how to fine-tune [Voxtral](https://hf.co/papers/2507.13264), Mistral AI's multimodal model that can perform speech transcription, translation, and audio understanding. Voxtral takes as input audio and/or text and outputs text, falling under the [`audio-text-to-text`](https://huggingface.co/models?pipeline_tag=audio-text-to-text) category of models.
"